iroh/src/magicsock.rs

```diff
@@ -2927,6 +2927,7 @@ impl Actor {
         self.update_direct_addresses(report);
     }
 
+    // TODO(frando): This never returns false.
```

Comment:
Might as well remove the return value instead of leave this as a TODO?

Reply:
done